opendbc/dbc/generator/SConscript

```diff
@@ -0,0 +1,4 @@
+Import('env')
+
+envDBC = env.Clone()
+envDBC.Command(None, "generator.py", "python3 $SOURCE")
```

Comment:
need to specify the dependencies and build products, otherwise it doesn't know when to build this (such as file changes)